### PR TITLE
fix(statistics) Use RTCRtpReceiver:getSynchronizationSources for audioLevels always

### PR DESCRIPTION
**Pull request by jallamsetty1**
Track based stats have been removed recently and RTCRtpReceiver:getSynchronizationSources has been available in the browsers for a while now.

### CODE REVIEW

**Comment by saghul** [comment]
This is not supported on RN since WebRTC doesn't implement it, it's implemented on the Chromium side. Will this be a problem?

**Comment by jallamsetty1** [comment]
> This is not supported on RN since WebRTC doesn't implement it, it's implemented on the Chromium side. Will this be a problem?

Actually this will fix the case for the newer releases of Chromium where the support for track statistics was dropped. Since RN WebRTC doesn't support this API, we were falling back to the legacy impl which wouldn't yield any results for newer versions of Chromium. RTCRtpReceiver:getSynchronizationSources has been supported since Chrome 73 so this change should fix this issue.

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Sorry, you lost me in the seconf half :-) I think we should be fine because we do disable audio levels on RN, I just wanted to make sure we wouldn't call that API on RN because it's not implemented.

**Comment by jallamsetty1** [comment]
> Sorry, you lost me in the seconf half :-) I think we should be fine because we do disable audio levels on RN, I just wanted to make sure we wouldn't call that API on RN because it's not implemented.

🤦‍♀️ now I get what you are saying. I can bring back the check and do nothing if the check fails. This should atleast help us get rid of the old getStats based impl. WDYT ?

File: JitsiConference.js [hunk 1]
```diff
@@ -531,7 +531,7 @@ JitsiConference.prototype._init = function(options = {}) {
     }
 
     // Generates events based on no audio input detector.
-    if (config.enableNoAudioDetection) {
+    if (config.enableNoAudioDetection && !RTC.options.disableAudioLevels) {
```
Comment (saghul) [inline JitsiConference.js:534]: Shouldn't this also be `!config.disableAudioLevels` ?

**Review by saghul — APPROVED** [review]

**Pull request merged**